IdentityCore/src/cache/MSIDMacTokenCache.m

```diff
@@ -378,15 +378,15 @@ - (BOOL)removeItemsWithKeyImpl:(MSIDCacheKey *)key
         userId = @"";
     }
     
-    NSMutableDictionary *userTokens = [self.cache[@"tokens"] objectForKey:userId];
+    NSMutableDictionary *userTokens = [(NSDictionary *)self.cache[@"tokens"] objectForKey:userId];
```

Comment:
self.cache is already declared as dictionary:

> @property (nonatomic) NSMutableDictionary *cache;

Why do we need to cast it to NSDictionary here?

Reply:
The compiler doesn't know the exact type returned by `self.cache[@"tokens"]`, and it's complaining multiple methods named `objectForKey`. And I think it should be `(NSMutableDictionary *)self.cache[@"tokens"]`, there is a check called `validateCache` to make sure Cache always a mutable dictionary in this file.